script_business_requirement_category/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
No README is ok: it is about the script itself which is not part of the module (this is a one-time script for a module in WIP so it is not like it was in production in many companies)